### PR TITLE
Popentry - Have the API return a db price if scraping fails

### LINKED ISSUE #19: Popentry - Have the API return a db price if scraping fails
https://trello.com/c/VZLvybv6/1238-2-popentry-have-the-api-return-a-db-price-if-scraping-fails

### PR DESCRIPTION
**Pull request by joaosa**
Closes #19

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "url": "git+https://github.com/Followprice/products.git"
   },
   "keywords": [],
-  "author": "Joao Andrade <jjadonotenter@gmail.com>",
+  "author": "Daniel Moreira <daniel.moureira@followprice.com>",
```
Comment (JoelAzevedo) [inline package.json]: The email is with moreira right? (not moureira) :)
daniel.moreira@followprice.com

Comment (JoelAzevedo) [inline package.json]: Or it was on purpose? Now that i see, your email was not right too... :)

Comment (joaosa) [inline package.json]: @JoelAzevedo it was a typo :). thanks!

**Comment by JoelAzevedo** [comment]
merged

**Pull request merged**